src/main/java/seedu/address/logic/commands/RemoveSuccess.java

```diff
@@ -0,0 +1,45 @@
+package seedu.address.logic.commands;
+
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.logic.Messages;
+import seedu.address.model.person.Person;
+import seedu.address.model.Model;
+
+import java.util.logging.Logger;
+
+import static java.util.Objects.requireNonNull;
+
+/**
+ * Performs the actual deletion of contact after user has been prompted for confirmation.
+ */
+public class RemoveSuccess extends RemoveConfirmation {
+
+    public static final String COMMAND_WORD = "yes";
+
+    public static final String MESSAGE_REMOVE_PERSON_SUCCESS = "Successfully Removed Contact: %1$s";
+
+    private final int DEFAULT_INDEX = 0;
+
+    public RemoveSuccess() {
+    }
```

Comment:
Are there places using the empty constructor as of now?

Reply:
Nope, will remove!